c_tests/tests/simple.c

```diff
@@ -36,16 +53,17 @@ void _yk_test(int i, int res) {
 int main(int argc, char **argv) {
   int res = 0;
   int loc = 0;
-  int i = 3;
+  int i = 5;
   NOOPT_VAL(res);
   NOOPT_VAL(i);
   while (i > 0) {
     yk_control_point(loc);
+    fprintf(stderr, "i=%d\n", i);
     res += 2;
     i--;
   }
+  // FIXME: unreachable due to aborting guard failure earlier.
```

Comment:
Instead of `FIXME` should we put some sort of abort / `err` / whatever in to enforce that?

Reply:
I've added an abort, but kept the `FIXME` as justification.

In 1a67a96.

Reply:
Good idea!